src/pages/IndividualFundraiserPage.js

```diff
@@ -80,57 +104,48 @@ const IndividualFundraiserPage = () => {
     );
 
     // Fundraiser Component
-    const Fundraiser = (
-        {
-            title,
-            description, 
-            createdDate, 
-            plannedEndDate, 
-            tags
-        }) => {
+    const Fundraiser = (props) => {
             return (
-            <div className='flex-row justify-left text-left pt-4 w-full'>
-                <div className='flex font-bold text-4xl'>{title}</div>
-                <div>
-                    <img src={placeholderImage} className="object-contain pt-12" alt="Logo" />
+            <div className='flex-row justify-left text-left w-full'>
+                <div className='flex font-bold text-4xl'>{props.title}</div>
+                <div className="object-contain pt-4">
+                    <img src={placeholderImage} className="object-contain pt-4" alt="Logo" />
                 </div>
-                <div className='flex text-sm italic pt-12'>Created On: { createdDate?.split("T")[0] }</div>
-                <div className='flex text-sm italic'>Ends On:  { plannedEndDate?.split("T")[0] }</div>
-                <div className='flex text-sm italic'>Tags:  {tags.map((tag) => (
+                <div className='flex text-sm italic pt-4'>Created On: {formatDate(props.createdDate)}</div>
+                <div className='flex text-sm italic'>Ends On:  {formatDate(props.plannedEndDate)}</div>
```

Comment:
{format(new Date(props.createdDate), 'MM/dd/yyyy')}

(from the comment above about the date format function